### PR TITLE
Add entry for PromCon EU 2019.

### PR DESCRIPTION
**Pull request by NightTsarina**

### CODE REVIEW

**Comment by mmuman** [comment]
Oh, another hit of this CI issue which doesn't send a User Agent I suppose…

**Comment by TheTincho** [comment]
> Oh, another hit of this CI issue which doesn't send a User Agent I suppose…

No.. It was really that the schedule was not yet live. That is fixed now, and also an error because the checks did not like a short link redirect

**Comment by Wilm0r** [comment]
Oh thanks crappy shithub UI for not actually sending you the comment above!

**Comment by NightTsarina** [comment]
Hi Wilmer!

> Oh thanks crappy shithub UI for not actually sending you the comment above two days ago when I wrote it!

Happened to me many times!

> I look forward to the day that shithub's code review UI is at least 1% as good as Critique's...

Hard same!

File: menu/promconeu-2019.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "end": "2018-11-08",
+        "metadata": {
+            "icon": "https://promcon.io/assets/favicons/apple-touch-icon.png",
+            "links": [
+                {
+                    "url": "https://promcon.io/2019-munich/",
+                    "title": "Website"
+                },
+                {
+                    "url": "https://www.google.de/maps/place/Google/@48.1429092,11.5387586,17z/data=!4m5!3m4!1s0x479e758b6989ce1b:0xdd1b393a488b0ebf!8m2!3d48.1429092!4d11.5409473",
```
[the remark this answers is not in the document]
Comment (NightTsarina) [reply menu/promconeu-2019.json]: will try!

**Comment by NightTsarina** [comment]
After a few attempts, I have found a format that works well both with google maps and osmand. PTAL?

**Pull request merged**